code/modules/cargo/packs.dm

```diff
@@ -2418,6 +2418,102 @@
 		cardpacktype = pick(subtypesof(/obj/item/cardpack))
 		new cardpacktype(C)
 
+//////////////////////////////////////////////////////////////////////////////
+////////////////////////// Pre-Built Machines ////////////////////////////////
```

Comment:
IMO  you should really switch the descriptions for the supply pack names and the item names. Usually, the supply pack names - as listed in the cargo console - are a bit more whimsical and advertisement-like, which seems to be the reverse of what's going on here.